README.md

```diff
@@ -145,11 +145,23 @@ Type: `String[]`<br>
 Default: `[]`
 
 ### timeout
-Timeout for step definitions.
+Timeout in milliseconds for step definitions.
 
 Type: `Number`<br>
 Default: `30000`
 
+### ignoreUndefinedDefinitions
```

Comment:
It's a _special_ option not recognized by cucumber, but it's used here
https://github.com/webdriverio/wdio-cucumber-framework/blob/master/lib/reporter.js#L100